src/o365/graph/commands/teams/teams-app-list.ts

```diff
@@ -0,0 +1,92 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import { CommandOption } from '../../../../Command';
+import { TeamsApp } from './TeamsApp'
+import { GraphItemsListCommand } from '../GraphItemsListCommand'
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  all?: boolean;
+}
+
+class TeamsAppListCommand extends GraphItemsListCommand<TeamsApp> {
```

Comment:
Following the naming convention, the class should be named `GraphTeamsAppListCommand`